databases/mysql-ob/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
@kdanW we don't need to commit this file.

We don't need to commit the `mysql-1.4.0.tgz` as well.

Reply:
Sorry, I have committed those files by mistake. will remove them